examples/cpp/visualize_openMVG_sfm_data/main.cpp

```diff
@@ -0,0 +1,96 @@
+        const auto id_pose = view->id_pose;
+
+        if (auto it = poses_container.find(id_pose); it != poses_container.end()) {
+            const auto& view_pose = poses_container.at(id_pose);
+            Eigen::Vector3d translation = -view_pose.rotation() * view_pose.center();
+
+            rerun::datatypes::Mat3x3 rr_rotation{
+                {(float)view_pose.rotation()(0, 0),
+                 (float)view_pose.rotation()(1, 0),
+                 (float)view_pose.rotation()(2, 0),
+                 (float)view_pose.rotation()(0, 1),
+                 (float)view_pose.rotation()(1, 1),
+                 (float)view_pose.rotation()(2, 1),
+                 (float)view_pose.rotation()(0, 2),
+                 (float)view_pose.rotation()(1, 2),
+                 (float)view_pose.rotation()(2, 2)}};
+
+            rerun::datatypes::Vec3D rr_translation{
+                (float)translation(0),
+                (float)translation(1),
+                (float)translation(2)};
+
+            rec.log(
+                "world/camera/"s + view_file_name,
+                rerun::archetypes::Transform3D(
+                    rerun::datatypes::TranslationAndMat3x3(rr_translation, rr_rotation, true)
+                )
+            );
+            const rerun::datatypes::Vec2D resolution{view->ui_width, view->ui_height};
+            rec.log(
+                "world/camera/"s + view_file_name,
+                rerun::archetypes::Pinhole::from_focal_length_and_resolution(
+                    sfm_data.GetIntrinsics().at(id_intrinsic)->getParams()[0],
+                    resolution
+                )
+            );
+            const auto& img = cv::imread(view->s_Img_path);
+            cv::cvtColor(img, img, cv::COLOR_BGR2RGB);
+            rec.log(
+                "world/camera/"s + view_file_name,
+                rerun::Image({img.rows, img.cols, img.channels()}, img.data)
+            );
+        }
+    }
+    const auto& landmarks = sfm_data.GetLandmarks();
+    std::vector<rerun::components::Position3D> points3d;
+    std::vector<rerun::components::KeypointId> track_ids;
+    std::unordered_map<uint32_t, std::vector<rerun::components::Position2D>> points2d_per_img;
+    for (const auto& landmark : landmarks) {
+        points3d.push_back(rerun::components::Position3D(
+            landmark.second.X(0),
+            landmark.second.X(1),
+            landmark.second.X(2)
+        ));
+        track_ids.push_back(landmark.first);
+        for (const auto& obs : landmark.second.obs) {
+            points2d_per_img[obs.first].push_back({obs.second.x(0), obs.second.x(1)});
+        }
+    }
+    rec.log("world/3Dpoints"s, rerun::archetypes::Points3D(points3d).with_keypoint_ids(track_ids));
+
+    for (const auto& view : sfm_data.views) {
+        rec.log(
+            "world/camera"s + view.second->s_Img_path,
+            rerun::archetypes::Points2D(points2d_per_img.at(view.second->id_view))
+        );
+    }
```

Comment:
Wouldn't it make more sense to put this on a timeline similar to the `structure_from_motion` python example?

Reply:
I fixed the code to work with your warning settings, and I will add another example to act like structure_from_motion python example, soon i will update the merge

Reply:
I dont really like to consider things as a timeline, since Structure From Motion is most of the time coming from unordered images set, but we can consider to add some. In my case I would love to see images added, and not consider as a single images/camera moving. Unordered image set can contain various camera type and resolutions.

Reply:
@Wumpf @pmoulon when you have time take a look at the updates.

Reply:
Right, from that pov timeline does _not_ make sense here.
In any case though the solution to this is surely not to duplicate the entire example Oo. If desired this can be a parameter

Reply:
I am agree with you and pierre that timeline does not make sense here, I just made the example to demonstrate more than one method to use openMVG with rerun, but I am  okay with removing the timeline example and I do not desire make it as a parameter to enable timeline because that will complicate the code and I intended to make it simple as possible to be expressive and readable